examples/pruning/tfkeras_integration.py

```diff
@@ -0,0 +1,149 @@
+"""
+Optuna example that demonstrates a pruner for tf.keras.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition
+using tf.keras and MNIST, where the architecture of the neural network
+and the parameters of optimizer are optimized.
+Throughout the training of neural networks,
+a pruner observes intermediate results and stops unpromising trials.
+
+You can run this example as follows:
+    $ python tfkeras_integration.py
+
+"""
+
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+import optuna
+from optuna.integration import TFKerasPruningCallback
+
+
+BATCHSIZE = 128
+CLASSES = 10
+EPOCHS = 20
+N_TRAIN_EXAMPLES = 60000
+STEPS_PER_EPOCH = int(N_TRAIN_EXAMPLES / BATCHSIZE / 10)
+VALIDATION_STEPS = 30
+
+
+def train_dataset():
+    # type: () -> tf.data.Dataset
+
+    ds = tfds.load('mnist', split=tfds.Split.TRAIN, shuffle_files=True)
+    ds = ds.map(lambda x: (tf.cast(x['image'], tf.float32)/255., x['label']))
+    ds = ds.repeat().shuffle(1024).batch(BATCHSIZE)
+    ds = ds.prefetch(tf.data.experimental.AUTOTUNE)
+
+    return ds
+
+
+def eval_dataset():
+    # type: () -> tf.data.Dataset
+
+    ds = tfds.load('mnist', split=tfds.Split.TEST, shuffle_files=False)
+    ds = ds.map(lambda x: (tf.cast(x['image'], tf.float32)/255., x['label']))
+    ds = ds.repeat().batch(BATCHSIZE)
+    ds = ds.prefetch(tf.data.experimental.AUTOTUNE)
+
+    return ds
+
+
+def create_model(trial):
+    # type: (optuna.trial) -> tf.keras.Model
+
+    # Hyper parameters to be tuned by Optuna.
```

Comment:
Please replace `Hyper parameters` with `Hyperparameters` to unify the terms used in this repository.

Reply:
I fixed it